tests/test_dual_net.py

```diff
@@ -20,19 +20,22 @@
 import go
 import preprocessing
 from tests import test_utils
+from absl import flags
 
 
 class TestDualNet(test_utils.MiniGoUnitTest):
     def test_train(self):
         with tempfile.TemporaryDirectory() as working_dir, \
                 tempfile.NamedTemporaryFile() as tf_record:
+            flags.FLAGS.model_dir = working_dir
```

Comment:
I see you can't use the test flagfile because you need to create a tmp directory...

Not sure I like putting model_dir as a FLAG.